core/src/executor/evaluate/error.rs

```diff
@@ -57,4 +57,10 @@ pub enum EvaluateError {
 
     #[error("subquery returns more than one row")]
     MoreThanOneRowReturned,
+
+    #[error("function requires formattable value: {0}")]
+    FunctionRequiresFormattableValue(String),
+
+    #[error("given specifier is not supported: {0}")]
```

Comment:
I added a new error type and this will be triggered when the invalid specifier is given.